frontend/utils/logo.ts

```diff
@@ -45,15 +43,11 @@ function splitWords (name: string) {
   return words
 }
 
-export function pickLogoLetter (name: string) {
+export function pickLogoLetter(name: string) {
   const fire = /[Ff]ire(base)?(-)?/
   const nameClean = name.replace(fire, '')
   const words = splitWords(nameClean)
   const firstLetter = words[0][0].toUpperCase()
 
   return firstLetter
 }
-
-export function daysAgo (date: Date) {
-  return formatDistanceToNow(date) + ' ago'
-}
```

Comment:
is this unused now? just double checking it wasn't removed by accident

Reply:
yeah, i don't think it's worth having a separate util function just to add "ago", heh. If we weren't using a node module for fuzzy dates and it were anymore complex, i could see it